.github/workflows/build.yml

```diff
@@ -98,7 +98,8 @@ jobs:
         env:
           RUST_BACKTRACE: full
         run: |
-          cargo test -- -- -Z unstable-options --format json | cargo2junit > results.xml
+          set -eo pipefail

```

Comment:
@Kethku looks like this flag fixes `not faling part`, but i don't know how to do the same for windows.

As for "not working json" part, i have one idea.

I've used my favorite debugging techniques "remove it and see if something changes". And i've removed everything from "main.rs".

And it fixed something.

Problem is: `cargo test -- --help` should pass arguments to libtest, as stated here: https://doc.rust-lang.org/cargo/commands/cargo-test.html

And in neovide, somehow, it does not pass anything until there is code in `main.rs`, so i thing problem is somewhere there.